packages/modules/viewer-react/src/services/BaseInitializer.ts

```diff
@@ -191,11 +191,7 @@ export class BaseInitializer {
       yield UiFramework.initialize(undefined);
 
       // initialize Presentation
-      yield Presentation.initialize({
-        presentation: {
-          activeLocale: IModelApp.localization.getLanguageList()[0],
```

Comment:
do we not need to provide a default for the active locale?

Reply:
It is optional and Presentation.initialize does the same what viewer was doing https://github.com/iTwin/itwinjs-core/blob/73187a864fe12a1ca44d38f799bc9a2d79dfa40a/presentation/frontend/src/presentation-frontend/Presentation.ts#L74